vsphere/cpi.yml

```diff
@@ -48,14 +48,22 @@
     password: ((vcenter_password))
     datacenters:
     - name: ((vcenter_dc))
-      vm_folder: ((vcenter_vms))
-      template_folder: ((vcenter_templates))
-      datastore_pattern: ((vcenter_ds))
-      persistent_datastore_pattern: ((vcenter_ds))
-      disk_path: ((vcenter_disks))
-      clusters:
-      - ((vcenter_cluster)): {}
+      vm_folder: ((vcenter_vm_folder))
+      template_folder: ((vcenter_templates_folder))
+      datastore_pattern: ((vcenter_ds_ephemeral))
+      persistent_datastore_pattern: ((vcenter_ds_persistent))
+      disk_path: ((vcenter_disk_path))
+      clusters: ((vcenter_cluster))
 
 - type: replace
   path: /cloud_provider/properties/vcenter?
   value: *vcenter
+
+- type: replace
```

Comment:
following 2 modification are not use by most. i dont think we should pull these ones in.